app/views/mailers/reservation_mailer/car_reservation_drivers_edited.html.erb

```diff
@@ -43,5 +43,13 @@
       To view/modify/cancel reservations, click here
       <%= link_to "LSA RideShare", welcome_pages_student_url %>
     </p>
+
+    <p>
+      <span style="color:red; font-weight:bold">Vehicle problems or breakdowns?</span>
+      <ul>
+        <li>Mon-Fri 8:00-12:30am call Garage Services at 734-764-2490</li>
```

Comment:
same review comment which is mentioned in _car_reservation_approved.html.erb_

Reply:
time fixed, code moved to partial